coinstacks/common/pulumi/src/rabbitmq.ts

```diff
@@ -0,0 +1,106 @@
+import * as pulumi from '@pulumi/pulumi'
+import * as k8s from '@pulumi/kubernetes'
+import { CustomResource } from '@pulumi/pulumi'
+import { rabbitmq, types } from './crds/types/rabbitmq'
+import { Config } from './index'
+
+export interface RabbitConfig {
+  cpuLimit: string
+  memoryLimit: string
+  replicaCount: number
+  storageClass: 'gp2' | 'hostpath' | 'standard'
+  storageSize: string
+}
+
+export async function deployRabbit(
+  app: string,
+  asset: string,
+  provider: k8s.Provider,
+  namespace: string,
+  config: Pick<Config, 'rabbit'>
+  ): Promise<CustomResource | undefined> {
+
+  const clusterName = `${asset}-rabbitmq`
+
+  const resources: pulumi.Input<types.input.rabbitmq.v1beta1.RabbitmqClusterSpecResourcesArgs> = {
+    limits: {
+      cpu: config.rabbit?.cpuLimit as pulumi.Input<string>,
```

Comment:
Can strings not be fed into an input?

Reply:
I think there might be a cleaner way to specify these values as `pulumi.Input`s, they cannot be strings.